terraform/sri-lanka/main.tf

```diff
@@ -0,0 +1,178 @@
+}
+
+#
+# slack alerts lambda
+#
+module "notify_slack" {
+  source  = "terraform-aws-modules/notify-slack/aws"
+  version = "3.2"
+
+  sns_topic_name       = "cloudwatch-to-slack"
+  slack_webhook_url    = var.slack_webhook_url
+  slack_channel        = var.slack_channel
+  slack_username       = var.slack_username
+
+  lambda_function_name = "cloudwatch-to-slack"
+}
+
+#
+# server configs
+#
+module "simple_server_sri_lanka_production" {
+  source                        = "../modules/simple_server"
+  deployment_name               = "sri-lanka-production"
+  database_vpc_id               = module.simple_networking.database_vpc_id
+  database_subnet_group_name    = module.simple_networking.database_subnet_group_name
+  ec2_instance_type             = "t2.medium"
+  ec2_ubuntu_version            = "20.04"
+  database_username             = var.production_database_username
+  database_password             = var.production_database_password
+  instance_security_groups      = module.simple_networking.instance_security_groups
+  aws_key_name                  = module.simple_aws_key_pair.simple_aws_key_name
+  server_vpc_id                 = module.simple_networking.server_vpc_id
+  https_listener_arn            = module.simple_networking.https_listener_arn
+  load_balancer_arn_suffix      = module.simple_networking.load_balancer_arn_suffix
+  host_urls                     = ["lk.simple.org", "api.lk.simple.org", "dashboard.lk.simple.org"]
+  create_redis_cache_instance   = true
+  create_redis_sidekiq_instance = true
+  create_database_replica       = true
+  server_count                  = 2
+  sidekiq_server_count          = 1
+  redis_param_group_name        = module.simple_redis_param_group.redis_param_group_name
+  enable_cloudwatch_alerts      = true
+  cloudwatch_alerts_sns_arn     = module.notify_slack.this_slack_topic_arn
+}
+
+module "simple_server_sri_lanka_demo" {
+  source                        = "../modules/simple_server"
+  deployment_name               = "sri-lanka-demo"
+  database_vpc_id               = module.simple_networking.database_vpc_id
+  database_subnet_group_name    = module.simple_networking.database_subnet_group_name
+  ec2_instance_type             = "t2.medium"
+  ec2_ubuntu_version            = "20.04"
+  database_username             = var.demo_database_username
+  database_password             = var.demo_database_password
+  instance_security_groups      = module.simple_networking.instance_security_groups
+  aws_key_name                  = module.simple_aws_key_pair.simple_aws_key_name
+  server_vpc_id                 = module.simple_networking.server_vpc_id
+  https_listener_arn            = module.simple_networking.https_listener_arn
+  load_balancer_arn_suffix      = module.simple_networking.load_balancer_arn_suffix
+  host_urls                     = ["api-demo.lk.simple.org", "api-staging.lk.simple.org", "dashboard-demo.lk.simple.org", "dashboard-staging.lk.simple.org", ]
```

Comment:
Let's avoid the use of `staging` if possible? No need to propagate any farther than it's already gotten?